config.json

```diff
@@ -177,7 +177,7 @@
         "name": "Gigasecond",
         "uuid": "fd7b62d4-266b-4e84-a526-bf3d47901216",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["dates"],
```

Comment:
Probably missing: immutability or whatever we call it in the main list.
Missing: type-conversion